tests/lib/prepare-project.sh

```diff
@@ -119,18 +121,15 @@ if [[ "$SPREAD_SYSTEM" == ubuntu-14.04-* ]]; then
     quiet apt-get install -y --force-yes apparmor libapparmor1 seccomp libseccomp2 systemd cgroup-lite util-linux
 fi
 
-quiet apt-get purge -y snapd
+distro_purge_package snapd
```

Comment:
It looks to me like the `distro_install cups` works on gentoo too, the distro_install might looks a bit ugly inside because it needs to do `emerge cups-foo-gentoo cups-bar-with-some-extras` but that appears to me not worse than springle the tests with this knowledge. I.e. the complexity/ugliness will appear somewhere, the question is if we show it in the test (which does not sound like a good deal given that the test is not about how cups is called on N distros) or if we show it in a central place. The other benefit is that when porting to a new distro there is just a single place to change, otherwise it will be a `git grep distro_install` exercise (which is not too bad so probably not a strong argument). Maybe I am missing something still but it seems like both approaches are fine but the centralized is less distracting in the tests itself. Plus I think it is something we can easily change later if it turns out that the approach we take is actually not pratcial.